sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/SessionReceiverManager.cs

```diff
@@ -234,6 +240,7 @@ private async Task CloseReceiver(CancellationToken cancellationToken)
                 // Always at least attempt to dispose. If this fails, it won't be retried.
                 await _receiver.DisposeAsync().ConfigureAwait(false);
```

Comment:
If the dispose fails, should `_receiver` and `_receiveTimeout` still have their state reset?

Reply:
Yes, similar to how we don't allow retrying close on sender/receiver, if the close here fails the processor will not retain a reference to the receiver and we expect the service to close the link after the idle timeout of 10 minutes.